playbooks/allocate_pubcloud.yml

```diff
@@ -58,7 +64,7 @@
           {% for instance in rax.success %}
           {{instance.name}} ansible_host={{instance.accessIPv4}} ansible_user=root
           {% endfor %}
-        dest: '{{lookup("env", "WORKSPACE")}}/rpc-gating/playbooks/inventory/hosts'
+        dest: '{{inventory_path}}/hosts'
```

Comment:
Thats needed because we insert a random string into inventory path for parallel runs, so we need to be able to override inventory_path.